lib/lib-grid/src/test/java/org/sagebionetworks/repo/model/grid/node/ObjectNodeTest.java

```diff
@@ -164,48 +161,16 @@ public void testAttemptInsertWithStartingNull() {
 		assertEquals(expected, node);
 	}
 
-	@Test
-	public void testAttemptInsertWithBothMapsNull() {
-		ObjectNode node = new ObjectNode().setId(ids.get(0));
-		node.setValue(null);
-
-		InsertObject change = new InsertObject().setObjectId(ids.get(0));
-		change.setMap(null);
-
-		// call under test
-		assertFalse(node.attemptInsert(change));
-		ObjectNode expected = new ObjectNode().setId(ids.get(0));
-		expected.setValue(null);
-		assertEquals(expected, node);
-	}
```

Comment:
It is no longer possible to create operations that are missing required properties, so these tests can be removed.